lib/active_admin/filters/active.rb

```diff
@@ -8,13 +8,21 @@ class Active
 
       def initialize(resource_class, params)
         @resource_class = resource_class
-        @params = params.respond_to?(:to_unsafe_h) ? params.to_unsafe_h : params
+        @params = normalize_params(params)
         @scope = humanize_scope
         @filters = build_filters
       end
 
       private
 
+      def normalize_params(params)
+        if params.kind_of?(HashWithIndifferentAccess)
```

Comment:
@varyonic , it doesn't for rails 4.2

Loading development environment (Rails 4.2.7.1)
2.2.0 :001 > params = ActionController::Parameters.new(name: 'Test')
 => {"name"=>"Test"} 
2.2.0 :002 > params === HashWithIndifferentAccess
 => false